netutils/nist.py

```diff
@@ -0,0 +1,130 @@
+"""Functions building NIST URLs from the os platform values."""
```

Comment:
There are 11 occurrences of either `is None` or `is not None` in this PR. I do not know the data, so they may all be needed, but could they be replaced by either one of these:

```python
if obj.get(value):
if not obj.get(value):
```

Reply:
Have updated these in the last commit.